resources-domain/resources-service/src/resource/resourceRoutesV0.ts

```diff
@@ -68,13 +47,13 @@ const resourceRoutes = (cloudSearchConfig: CloudSearchConfig) => {
     },
   );
 
-  router.post('/search-es', async (req, res) => {
+  router.post('/search', async (req, res) => {
     const { limit, start } = req.query;
-    const { q, filters = {} } = req.body;
 
-    const referrableResources = await searchResourcesEs(<AccountSID>req.accountSid, {
-      filters: filters as Record<string, string | boolean | number>,
-      q: q as string,
+    const referrableResources = await searchResources(<AccountSID>req.accountSid, {
+      filters: {},
+      generalSearchTerm: '',
+      ...req.body,
```

Comment:
Is there a reason to pass empty `filters` and `generalSearchTerm` here, instead than abstracting those in the `searchResources` params?

Reply:
They are passed in the `searchResources` params - they are just defaults. If the request body specifies them they are overwritten